integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5128,6 +5132,7 @@ msgid "or"
 msgstr "oder"
 
 #: cms/templates/languages/language_form.html
+<<<<<<< HEAD
```

Comment:
Forgot a `./tools/resolve_translation_conflicts.sh`?

Reply:
I forgot to add the change while rebasing 😓 It'll be fixed!